addons/website_event_exhibitor/static/src/components/exhibitor_connect_closed_dialog/exhibitor_connect_closed_dialog.xml

```diff
@@ -4,7 +4,10 @@
   <t t-name="website_event_exhibitor.ExhibitorConnectClosedDialog">
     <Dialog size="'md'" header="false" footer="false" >
         <div class="o_wesponsor_js_connect_modal_main container">
-            <div class="row mt-2">
+            <div class="row">
+                <div class="col-12 px-0 mb-2">
+                    <button class="btn btn-link text-decoration-none float-end fa fa-times p-1" t-on-click="() => this.props.close()"/>
```

Comment:
Is "float-end" how those are usually defined? I feel like we tend to use "align-items-end" on the contained more.

Reply:
@awa-odoo Well, actually it looks like the button has been fixed in the meantime in https://github.com/odoo/odoo/pull/163105

Reply:
Ah, indeed, I know the reviewer, a very good person 🥸 

Thanks for the heads-up!

Reply:
@awa-odoo No issue, I was surprised to see two crosses after rebasing ^^ Twice's the charm!

There was also a small fix regarding the badge display, so I've updated the PR :)